README.md

```diff
@@ -1,71 +1,98 @@
-[![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline "Publish Pipeline Badge")](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
+[![Validation Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=ValidationPipeline 'Validation Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=337)
+[![Publish Pipeline Badge](https://img.shields.io/endpoint?url=https://winget-pme.azurefd.net/api/GetServiceComponentStatusBadge?component=PublishPipeline 'Publish Pipeline Badge')](https://dev.azure.com/ms/winget-pkgs/_build?definitionId=338)
 [![Github Status](https://img.shields.io/endpoint?url=https://api.bittu.eu.org/github-status-badge-endpoint)](https://www.githubstatus.com)
 
 This repository contains the manifest files for the **Windows Package Manager** default source. You are highly encouraged to submit manifests for your favorite application.
+
 > Note: At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
 
 The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).
 
 # Submitting a Package
+
 To submit a package to this repository, you should follow these steps:
-1) Follow the [Contributing](#Contributing) guidelines below.
-2) [Author](AUTHORING_MANIFESTS.md) a Manifest.
-3) [Test](#test-your-manifest) your manifest.
-4) [Submit](#submit-your-pr) your pull request (PR).
-5) Respond to any feedback in your PR.
+
+1. Follow the [Contributing](#Contributing) guidelines below.
+2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
+3. [Test](#test-your-manifest) your manifest.
+4. [Submit](#submit-your-pr) your pull request (PR).
+5. Respond to any feedback in your PR.
 
 > Note: Please check the package's manifest you intend to submit does not already exist in the repository, and there are no open PRs for it in order to avoid duplicates.
 
 ## Authoring a Manifest
 
-You may either use the [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create), the [YamlCreate](#using-the-yamlcreateps1) PowerShell script, or you can craft a manifest manually following our [authoring guidelines](AUTHORING_MANIFESTS.md).
+There are some tools available to help you author a manifest.
+
+- [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create)
+- [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)
```

Comment:
I agree. We're leading with "wingetcreate" since it's a product we develop and maintain. It's designed with CI/CD scenarios in mind. YAMLCreate would be next as it's also in the Microsoft organization.

I'd suggest after those something like:

## Alternatives
Other open-source tooling has been developed by third parties. 
* [Option 1](https://url.com)
* [Option 2](https://url.com)

---
I'd rather avoid any specific descriptions for those projects, and I'd list them in alphabetical order. We need to make sure we're not endorsing or otherwise giving any impression of favoritism.